hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java

```diff
@@ -114,10 +114,14 @@ public void handle(@NonNull final HandleContext handleContext) throws HandleExce
         final var file = maybeFile.get();
         validateFalse(file.deleted(), FILE_DELETED);
 
-        final var fees = handleContext.feeCalculator(SubType.DEFAULT).legacyCalculate(sigValueObj -> {
-            return new FileUpdateResourceUsage(fileOpsUsage)
-                    .usageGiven(fromPbj(handleContext.body()), sigValueObj, fromPbj(file));
-        });
+        // Feature: This is not working as expected. This should be reworked and added support for a system file or
+        // privileged users. When this is done we can turn on the following tests in  ThrottleDefValidationSuite -
+        // updateWithMissingTokenMintGetsWarning, throttleUpdateWithZeroGroupOpsPerSecFails, and
+        // throttleUpdateRejectsMultiGroupAssignment
+        final var fees = handleContext
+                .feeCalculator(SubType.DEFAULT)
+                .legacyCalculate(sigValueObj -> new FileUpdateResourceUsage(fileOpsUsage)
+                        .usageGiven(fromPbj(handleContext.body()), sigValueObj, fromPbj(file)));
 
         handleContext.feeAccumulator().charge(handleContext.payer(), fees);
```

Comment:
Just an info, This should be moved to `calculateeFees()` method also. @povolev15 is working on it.